pipelines/split/bin.yaml

```diff
@@ -0,0 +1,40 @@
+name: Split executable files
+
+needs:
+  packages:
+    - busybox
+
+inputs:
+  package:
+    description: |
+      The package to split executable files from
+    required: false
+
+pipeline:
+  - runs: |
+      pkgname="${{context.name}}"
+      pkgnobin="${pkgname%%-bin}"
+      pkgnobindir="${{targets.outdir}}/$pkgnobin"
+
+      if [ -n "${{inputs.package}}" ]; then
+        PACKAGE_DIR="${{targets.outdir}}/${{inputs.package}}"
+      elif [ "$pkgname" != "$pkgnobin" ] && [ -d "$pkgnobindir" ]; then
+        PACKAGE_DIR="$pkgnobindir"
+      else
+        PACKAGE_DIR="${{targets.outdir}}/${{targets.destdir}}"
+      fi
+
+      if [ "$PACKAGE_DIR" == "${{targets.contextdir}}" ]; then
+        echo "ERROR: Package can not split files from itself!" && exit 1
+      fi
+
+      cd "$PACKAGE_DIR" || exit 0
+
+      for i in usr/bin; do
```

Comment:
shellcheck would tell you that https://www.shellcheck.net/wiki/SC2043 .
I dont care about that too much, but there is value in shellcheck.